react-frontend/src/components/Charts.tsx

```diff
@@ -112,7 +168,21 @@ function Charts({
 					if (response.status != 200) {
 						console.warn(response.data.error);
 					}
-					setWolframResults(response.data.wolfram_says);
+					// keys are "closed_forms" and "metadata" where metadata has entries of the form: ["text": "", "links": {"url":"...", ...}]
+					// "links" can also be an array of objects
+					let wolframData = response.data.wolfram_says.closed_forms;
+					// add link from metadata if present
+					// add description from metadata if present
+					setWolframResults(wolframData);
+					if (typeof response.data.wolfram_says.metadata != 'undefined') {
+						if (Array.isArray(response.data.wolfram_says.metadata)) {
```

Comment:
because it can be either an array of objects or an object 🤢

Reply:
An object is array of size 1 ... right?

Reply:
😆 not equivalent in JSON, tho